src/openqa_client/client.py

```diff
@@ -377,3 +377,30 @@ def get_jobs(
             # there as it only considers the jobs queried.
             jobdicts = self.find_clones(jobdicts)
         return jobdicts
+
+    def get_latest_build(
+        self,
+        group_id: int,
+        all_passed: bool = True,
+        sort_key: Callable = int,
+    ) -> str:
+        """Identify latest build number in target Job Group
+
+        Args:
+            group_id (int): Job Group ID
+            all_passed (bool, optional): Controls whether just last build will be selected
+                or the last with all jobs passed. Defaults to True.
+            sorted_key (Callable, optional): To find the latest build we need to order the builds.
+                The specified callback will be used in the sorted function to sort builds.
+                Defaults to int.
+
+        Returns:
+            str: string representation of last BUILD
+        """
+        resp = self.openqa_request("GET", f"/group_overview/{group_id}.json")
```

Comment:
Rather than retrieving that webui url, can you instead try the `/job_groups/id/build_results` API endpoint instead? See https://progress.opensuse.org/issues/152939

Reply:
done